### DISCUSSION TITLE
0.23 Authentication section in admin settings missing

### DISCUSSION THREAD

**Discussion by petrm** [pocketbase/pocketbase#5938] [Q&A] [answered]
Hi,
after upgrading to 0.23.0 release I don't see the "Authentication" section of the settings page anymore. Building from source results in the same.
Is this expected?

**Comment by ganigeorgiev** [comment]
Sorry, I forgot to mention it. It is expected because the mail templates, token options and OAuth2 provider configurations are now under the specific auth collection options (_aka. you'll find the configurations in Dashboard > Collections > :authCollection > Edit collection > Options tab_).

**Comment by petrm** [comment]
I must be looking in the wrong place?
![image](https://github.com/user-attachments/assets/ae05531f-9514-4c2e-b3ff-e0cafb6afb57)

**Reply by ganigeorgiev** [reply, marked as answer]
An auth collection is for example the `_superusers` and the default `users` collection (if you have such).

You can inspect the `users` collection in the live demo - https://pocketbase.io/demo/

**Reply by petrm** [reply]
I found it, thanks. That was really well hidden.

**Comment by thatdudemanguy** [comment]
Hopping on this thread. 

How do we actually add an 0auth provider now, for instance Google? The section on [authentication](https://pocketbase.io/docs/authentication/#authenticate-with-oauth2) currently refers to the settings page (which is removed in 0.23.0):
`Once you have obtained the Client Id and Client Secret, you can enable and configure the provider from your PocketBase admin settings page.`

Cannot seem to figure this change out on my own.

**Reply by ganigeorgiev** [reply]
I'll update the documentation a little bit later (_Done_). The OAuth2 and other auth method options are now part of the specific auth collection options, aka. go to _"Dashboard > Collections > {:yourAuthCollection} > Edit collection (settings cogwheel) > Options > OAuth2"_.

**Reply by thatdudemanguy** [reply]
Ooh I see, gotcha!

Thanks for the help and the insane reaction time.
